cloudFerry.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
Please rename file to `cloudferry.py`. The uppercase 'F' may result in troubles on non case-sensitive file systems.